### PR TITLE
change: used `lua-resty-jsonschema` instead of `lua-rapidjson`.

### PR DESCRIPTION
**Pull request by membphis**
lua-resty-jsonschema is faster and easy to compile.

todo items:

- [x] doc: install

- [x] remove rapidjson and add lua-resty-jsonschema in copyright file.

### CODE REVIEW

File: t/admin/global-rules.t [hunk 1]
```diff
@@ -218,8 +218,8 @@ GET /t
 --- request
 GET /t
 --- error_code: 400
---- response_body
-{"error_msg":"invalid configuration: invalid \"additionalProperties\" in docuement at pointer \"#\/host\""}
+--- response_body_like eval
+qr/{"error_msg":"invalid configuration:/
```
Comment (moonming) [inline t/admin/global-rules.t]: we lost the detail error info?

File: rockspec/apisix-dev-1.0-0.rockspec [hunk 2]
```diff
@@ -32,6 +31,7 @@ dependencies = {
     "luafilesystem = 1.7.0-2",
     "lua-tinyyaml = 0.1",
     "iresty-nginx-lua-prometheus = 0.20190917",
+    "lua-resty-jsonschema = 0.1",
```
Comment (moonming) [inline rockspec/apisix-dev-1.0-0.rockspec]: remove `rapidjson` and add `lua-resty-jsonschema` in copyright file.

File: lua/apisix/core/schema.lua [hunk 3]
```diff
@@ -1,32 +1,24 @@
...
 local function create_validator(schema)
-    local sd = schema_doc(schema)
-    local validator = schema_validator(sd)
-
-    -- need to cache `validator` and `sd` object at same time
-    return {validator, sd}
+    -- local code = jsonschema.generate_validator_code(schema, opts)
+    -- local file2=io.output("/tmp/2.txt")
+    -- file2:write(code)
+    -- file2:close()
```
Comment (moonming) [inline lua/apisix/core/schema.lua]: remove debug codes.

File: t/admin/global-rules.t (hunk 1 above)
Comment (membphis) [reply t/admin/global-rules.t]: different json schema will throw different error message, we do not want to check the detail info.

File: lua/apisix/core/schema.lua (hunk 3 above)
Comment (membphis) [reply lua/apisix/core/schema.lua]: we can remove it after a while(maybe 1 month)

File: rockspec/apisix-dev-1.0-0.rockspec (hunk 2 above)
Comment (membphis) [reply rockspec/apisix-dev-1.0-0.rockspec]: add todo list

**Pull request merged**